src/plugins/expressions/common/expression_types/datatable.ts

```diff
@@ -59,7 +59,7 @@ export interface Datatable {
   rows: DatatableRow[];
 }
 
-interface SerializedDatatable extends Datatable {
+export interface SerializedDatatable extends Datatable {
```

Comment:
I don't see this being used anywhere in the PR, so why the export?

Reply:
@lizozom tsc is failing in this PR without having this interface exported, as it cannot implicitly guess the interface being used. I dont claim to fully understand what was happening but exporting this interface solves it.